### PR TITLE
Add gci to seperate goimports to three parts

### PR DESCRIPTION
**Pull request by wuyingjun-lucky**
Signed-off-by: wuyingjun <wuyingjun_yewu@cmss.chinamobile.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Add gci to seperate goimports to three parts
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks!

/lgtm

**Comment by prodanlabs** [comment]
great!

/lgtm

File: .golangci.yml [hunk 1]
```diff
@@ -62,3 +68,12 @@ issues:
   - EXC0012
   - EXC0013
   - EXC0014
+  exclude-rules:
+    # Exclude 'cmd/agent/main.go' and 'cmd/controller-manager/controller-manager.go'
+    # because 'sigs.k8s.io/controller-runtime/pkg/metrics' package could be set before 'k8s.io/apiserver/pkg/server' .
```
Comment (RainbowMango) [inline .golangci.yml:73]: `could be set` --> `should` be set?

**Review by RainbowMango — COMMENTED** [review]
Thanks and I'd like to invite the gci author to take a look.
cc @daixiang0

File: .golangci.yml (hunk 1 above)
Comment (wuyingjun-lucky) [reply .golangci.yml]: done

**Review by daixiang0 — APPROVED** [review]
lgtm

**Comment by RainbowMango** [comment]
/lgtm
/approve

Thanks @daixiang0 @wuyingjun-lucky

**Pull request merged**